deploy/examples/subvolumegroup.yaml

```
@@ -0,0 +1,9 @@
+---
+apiVersion: ceph.rook.io/v1
+kind: CephFilesystemSubVolumeGroup
+metadata:
+  name: my-subvolumegroup
+  namespace: rook-ceph # namespace:cluster
+spec:
+  # volumeName is the name of the CephFilesystem CR
```

Maybe a slightly expanded explanation would help?

```suggestion
  # volumeName is the metadata name of the CephFilesystem CR where the subvolume group will be created
```